backend/btrixcloud/basecrawls.py

```diff
@@ -258,6 +272,57 @@ async def _delete_crawl_files(self, crawl, org: Organization):
 
         return size
 
+    async def _resolve_automated_crawl_refs(
+        self,
+        crawl: BaseCrawlOut,
+        org: Optional[Organization],
+        add_first_seed: bool = True,
+    ):
+        """Resolve running crawl data"""
+        # pylint: disable=too-many-branches
+        config = await self.crawl_configs.get_crawl_config(
+            crawl.cid, org, active_only=False
+        )
+
+        if config:
+            if not crawl.name:
+                crawl.name = config.name
+
+            if not crawl.description:
+                crawl.description = config.description
+
+            if config.config.seeds:
+                if add_first_seed:
+                    first_seed = config.config.seeds[0]
+                    crawl.firstSeed = first_seed.url
+                crawl.seedCount = len(config.config.seeds)
+
+        if hasattr(crawl, "profileid") and crawl.profileid:
+            crawl.profileName = await self.crawl_configs.profiles.get_profile_name(
+                crawl.profileid, org
+            )
+
+        user = await self.user_manager.get(crawl.userid)
+        if user:
+            crawl.userName = user.name
+
+        # if running, get stats directly from redis
+        # more responsive, saves db update in operator
+        if crawl.state in (
```

Comment:
We should move the list of running states to basecrawls to avoid duplications / possible errors if we add more states. Should we just move _resolve_crawl_refs() to be here and avoid further duplication?

Reply:
Yup, good points! I'll move both to basecrawls